pkg/networkplugin-syncer/handlers/ovn/router_ovn_cluster_north.go

```diff
@@ -16,8 +16,8 @@ const (
 	ovnClusterSubmarinerSwPort = "ovn_cluster_subm_lsp"
 	ovnClusterSubmarinerMAC    = "00:60:2f:10:01:02"
 	ovnClusterSubmarinerNET    = ovnClusterSubmarinerIP + "/29"
-	ovnClusterSubmarinerIP     = "169.254.34.2"
-	ovnRoutePoliciesPrio       = 10
+	ovnClusterSubmarinerIP     = "169.254.254.2"
+	ovnRoutePoliciesPrio       = 10000
```

Comment:
```suggestion
	ovnRoutePoliciesPrio       = 20000
```
I've seen ovn uses up to 10000, we also want to prepare a meeting with them, and probably reserve some IPs, and priorities in OVNKubernetes codebase for us to use.

Reply:
Done